src/test/java/com/znsio/templates/jsonPlaceholderTemplates.feature

```diff
@@ -0,0 +1,36 @@
+@template
+Feature: API tests for https://jsonplaceholder.typicode.com/
+
+  Background:
+    Given url env.fetchPostsAndAlbumsUrl
+    And print "fetchPostsAndAlbumsUrl: " + env.fetchPostsAndAlbumsUrl
+
+  @t_getPosts
+  Scenario: Get list of posts for userId 1
+    Given def path = "/posts"
+    * def queryParam = {userId : '1'}
```

Comment:
Why we are declaring userId here in template?  What if any other scenario wants to run same template with different userId?